### DISCUSSION TITLE
Error in build

### DISCUSSION THREAD

**Discussion by Thecode764** [pradyunsg/furo#791] [Potential Issue]
hi my error is
Extension error:
Could not import extension furo (exception: No module named 'furo')

**Comment by pradyunsg** [comment]
Hi @Thecode764! You likely did not install Furo or are trying to use an installation of Sphinx that doesn't have Furo installed.

**Reply by Thecode764** [reply]
hello
thank you very much but i hosting my read the docs documation on github
how to install furo in github?
